src/workers/parser/getSymbols.js

```diff
@@ -121,6 +122,12 @@ function extractSymbols(source: Source) {
         });
       }
 
+      if (t.isJSXElement(path)) {
+        setJSXMetaData(source, true);
```

Comment:
`setJSXMetaData` unfortunately won't work because you can't call reducer methods directly.

I recommend adding a `jsx` field to the object that is returned. That way, we'll be able to look up the jsx value wherever symbols are used